.travis.yml

```diff
@@ -9,6 +9,9 @@ before_install:
   - export CHROME_BIN=chromium-browser
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
+  - which python
+  - python --version
```

Comment:
I was playing a hunch because `node-gyp` was the part of the process that was failing. And `node-gyp` requires python ([node-gyp install notes](https://github.com/nodejs/node-gyp#installation)):

> On Unix:
> * `python` (`v2.7` recommended, `v3.x.x` is _**not**_ supported)
> * `make`
> * A proper C/C++ compiler toolchain, like GCC

Turns out it's running the correct version. I'll do some more tests this weekend. BTW, I've also tested this in an Ubuntu VM and I can't get it to error in the same way during `npm install`. So it's a little difficult for me to reproduce this without using travis.